charts/node/Chart.yaml

```diff
@@ -2,7 +2,7 @@ apiVersion: v2
 name: node
 description: A Helm chart to deploy Substrate/Polkadot nodes
 type: application
-version: 4.0.2
+version: 4.0.3
```

Comment:
please bump the minor version

Reply:
Should be minor instead of fix so: 4.1.0